samples/client/petstore/scala-sttp/src/main/scala/org/openapitools/client/model/ApiResponse.scala

```diff
@@ -0,0 +1,22 @@
+/**
+ * OpenAPI Petstore
+ * This is a sample server Petstore server. For this sample, you can use the api key `special-key` to test the authorization filters.
+ *
+ * The version of the OpenAPI document: 1.0.0
+ * 
+ *
+ * NOTE: This class is auto generated by OpenAPI Generator (https://openapi-generator.tech).
+ * https://openapi-generator.tech
+ * Do not edit the class manually.
+ */
+package org.openapitools.client.model
+
+import org.openapitools.client.core.ApiModel
+
+case class ApiResponse (
```

Comment:
Missing comments for public API (is this a part of generator API or generated (pets) API?)

Reply:
added